crates/ruff/src/rules/flake8_todos/rules/todos.rs

```diff
@@ -356,10 +356,10 @@ fn static_errors(
             if let Some(end_index) = trimmed.find(')') {
                 TextSize::try_from(end_index + 1).unwrap()
             } else {
-                trimmed.text_len()
+                TextSize::try_from(trimmed.find(' ') + 1).unwrap()
             }
         } else if trimmed.starts_with('@') {
-            trimmed.text_len()
+            TextSize::try_from(trimmed.find(' ') + 1).unwrap()
```

Comment:
In this case, we do need to find the whitespace. To generalize across all whitespace, we can do something like `TextSize::try_from(trimmed.find(char::is_whitespace))` to find the offset of the first whitespace in `trimmed`

Reply:
ok, so `TextSize::try_from(trimmed.find(char::is_whitespace)).unwrap()`? (Sorry, I am not a rust developer)